integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/tpcds/TpcdsLikeBench.scala

```diff
@@ -33,19 +33,21 @@ object TpcdsLikeBench extends Logging {
    * @param spark The Spark session
    * @param query The name of the query to run e.g. "q5"
    * @param iterations The number of times to run the query.
+   * @param resultFilenameStub Optional name for the generated JSON summary file.
```

Comment:
I'm assuming Stub here was intentional because I think its actually only part of the file name, we may want to clarify the description to say prefix for the...

Reply:
Thanks. I renamed to `summaryFilePrefix` to disambiguate from the output arguments that refer to the output of the query itself. I also updated the javadocs.